climada/util/api_client.py

```diff
@@ -486,3 +533,166 @@ def purge_cache(self, local_path):
         """
         dlf = Download.get(Download.path==str(local_path.absolute()))
         dlf.delete_instance()
+
+    def get_hazard(self, hazard_type, dump_dir=SYSTEM_DIR, **kwargs):
```

Comment:
should we maybe set a limit to the number of datasets that are downloaded in these functions by default? if you simply do get_hazard() or get_exposures() you get thousands of datasets downloaded it may be a bit dangerous for a new user that doesn't know anything about these functions?